pkgs/stdenv/adapters.nix

```diff
@@ -234,6 +234,30 @@ rec {
       });
     });
 
+  useLibsFrom = modelStdenv: targetStdenv:
+    let
+      ccForLibs = modelStdenv.cc.cc;
+      cc = pkgs.wrapCCWith {
+        /* NOTE: cc.cc is the unwrapped compiler. Should we respect the old
+         * wrapper instead? */
+        cc = targetStdenv.cc.cc;
```

Comment:
This is the part I was asking about in https://matrix.to/#/%23stdenv%3Anixos.org/%24jhoyDDs4_NDkhlkmBihy57cv9OSaRkg8zO17cTbr5-0?via=someonex.net&via=matrix.org&via=kde.org&via=matrix.mit.edu